test/replication/qsync_basic.test.lua

```diff
@@ -255,6 +256,7 @@ box.space.sync:count()
 test_run:cmd('switch default')
 for i = 1, 100 do box.space.sync:delete{i} end
 test_run:cmd('switch replica')
+test_run:wait_lsn('replica', 'default')
```

Comment:
Same about this `wait_lsn`. The quorum's still 2 on master and no mvcc. Replica should see all 100 deletes as soon as they complete.